tutorial/about-tutorial.md

```diff
@@ -48,7 +48,7 @@ Make sure you meet the following prerequisites before starting the tutorial step
 
 * API credentials for the {company} v3.5 API.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is, conceptually.
```

Comment:
If you agree with the review comment above for swapping "about" with "of" then make the same change here.

Reply:
Sure.

Reply:
Will you push another change for this suggestion @Dishebh ?

Reply:
Same here. Already committed the change.